src/content/en/tools/workbox/_shared/contributors.html

```diff
@@ -0,0 +1,181 @@
+{% framebox height="auto" %}
```

Comment:
Why is it in a framebox? I don't see any JS. You could do it simply as an html or md include.

Reply:
Will remove but I think I added this so `_index.yaml` would be happy.

Reply:
OH! Fudge. Imports into `_index.yaml` are a PITA! Uh, let's remove it for now, but we may need to be extra careful on this one. :/